docs/reference/UDS Core/functional-layers.md

```diff
@@ -46,3 +46,37 @@ packages:
     repository: ghcr.io/defenseunicorns/packages/uds/core-backup-restore
     ref: 0.29.0-upstream
 ```
+
+## Layer Selection
+
+Layer selection will always be deployment-specific but below are guidelines for what layers to consider for your deployment.  The *required* layer guidance follows the [Big Bang Conformant Stack](https://repo1.dso.mil/big-bang/product/bbtoc/-/blob/master/policy/conformance.md?ref_type=heads) though if you are not bound by that document you may need to tweak them for your needs - UDS Core Base is the only *technical* layer required to install most UDS Packages.
```

Comment:
On the last line that's mostly accurate, not sure if it's worth calling out that some layers require identity (monitoring and runtime-security) and logging provides no UI without monitoring (maybe not necessary to say, still an API provided?).

Reply:
Made some changes to note that the monitoring and runtime security layers need IDAM and reworded logging / monitoring to make it more clear that one was "backend" and the other "frontend"